packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -747,6 +765,7 @@ class Splitter extends (CollectionWidget as any) {
 
     switch (name) {
       case 'allowKeyboardNavigation':
+        this._updateNestedSplitterOption(name, value);
```

Comment:
Let's safe order - change for parent component and only after this - change correspond options for nested components.